README.md

```diff
@@ -3,6 +3,28 @@ Pi4J :: Java I/O Library for Raspberry Pi :: Devices
 
 Collection of devices implemented for use with Pi4J V2.
 
+
+  __  __ _               _   _          
```

Comment:
Probably makes sense to add triple quotes to preserve the ascii art in md?

Reply:
PR109